docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -59,19 +59,30 @@ image::images/view-alert-details.png[Shows the Event Renderer button, 200]
 
 The Alert details flyout appears and offers several options for viewing alert details:
 
-* *Summary*: Shows an aggregated view of alert details. Alerts that have been enriched with `threat.indicator` data also display the *threat summary* section, which is an additional section located beneath the alert summary. In the *threat summary* section, you can view mapped data for the following `threat.indicator` subfields:
-** `matched.field`
-** `matched.type`
-** `source (threat.indicator.provider)`
-** `first_seen`
-** `last_seen`
+* *Summary*: Shows an aggregated view of alert details, plus two types of threat intelligence data: indicator rule enrichments and investigation time enrichments.
+** *Indicator rule enrichments*: As the name suggests, indicator rule enrichments only display for alerts produced by indicator match rules. You can spot an indicator rule enrichment by checking for fields pre-fixed by the {es-sec} icon and appended with source event data, if that information is available. Indicator rule enrichments are created when rule indicators are met and mapped index fields are produced. An alert can have one or more indicator rule enrichment and the same indicator match can display multiple times.
```

Comment:
in the first sentence we could reconsider the word `display` as it sounds like they could have existed somewhere else but `display` only for indicator match rule alerts, when in reality they don't exist otherwise. In the second sentence it sounds like we would have displayed the icon anyway and `append source event data` only if it exists, instead we could just say something along the lines of `when there is a threat match you can see the matched.field and matched atomic next to the {es-sec}  icon`. The next sentence sounds a bit awkward, we could say something like `Indicator rule enrichments are created when threat indicators are matched with source events according to the conditions outlined in the relevant indicator match rule definition`. From the next sentence we can remove `and the same indicator match can display multiple times` as they are not supposed to be the same match, just looks the same in the summary area, we have a ticket to improve the UX around that.